spec/command/package_spec.rb

```diff
@@ -128,6 +128,30 @@ module Pod
                                 BBUNikePlusTag }
       end
 
+      it "does not mangle standalone preserve symbols" do
+        SourcesManager.stubs(:search).returns(nil)
+
+        command = Command.parse(%w{ package spec/fixtures/cxa-symbols.podspec --preserve-symbols=__cxa_throw})
+        command.run
+
+        lib = Dir.glob("cxa-symbols-*/ios/cxa-symbols.framework/cxa-symbols").first
+        symbols = Symbols.symbols_from_library(lib).uniq.sort.reject { |e| e =~ /Podcxa_symbols|PodsDummy_cxa_symbols/ }
+        symbols.should == %w{ BBUNikePlusActivity BBUNikePlusSessionManager
+                              BBUNikePlusTag __cxa_throw }
+      end
+      
+      it "does not mangle preserve symbol patterns" do
+        SourcesManager.stubs(:search).returns(nil)
+
+        command = Command.parse(%w{ package spec/fixtures/cxa-symbols.podspec --preserve-symbols=__cxa_throw,^KSCrash$})
+        command.run
+
+        lib = Dir.glob("cxa-symbols-*/ios/cxa-symbols.framework/cxa-symbols").first
+        symbols = Symbols.symbols_from_library(lib).uniq.sort.reject { |e| e =~ /Podcxa_symbols|PodsDummy_cxa_symbols/ }
```

Comment:
Full blown ruby regexp might be an overkill for command line options and bash "globs" should be enough. But I can't find proper gem to translate globs syntax to regexps.